### PR TITLE
generated and exported typescript types for @airbrake/browser package

### PR DESCRIPTION
**Pull request by HelixHEX**
Installed dts rollup plugin to include export typescript typings.

(Note: types are exported to the folder /packages/browser/types/index.d.ts)

### CODE REVIEW

**Comment by kyrylo** [comment]
Hey @HelixHEX, thanks for the PR. Do you think it makes sense to do the same for `@airbrake/node`?

**Comment by HelixHEX** [comment]
> Hey @HelixHEX, thanks for the PR. Do you think it makes sense to do the same for `@airbrake/node`?

It does make sense. It ensures that all packages are type safe out the box. 

If you want to reduce the package size, some package managers export types to a separate package (for example: @types/airbrake-browser) but this requires much more configuration which is why I defaulted to exporting types in one package.

File: packages/browser/rollup.config.js [hunk 1]
```diff
@@ -19,15 +20,24 @@ function umd(cfg) {
       sourcemap: true,
       name: 'Airbrake',
     },
-    cfg,
+    cfg
```
Comment (kyrylo) [inline packages/browser/rollup.config.js:23]: Please, undo this.

**Review by kyrylo — COMMENTED** [review]
The instructions for `rollup-plugin-dts` state that we need to provide `typings` in `package.json`. I cannot see that in this PR. Could you, please, explain?

**Comment by kyrylo** [comment]
Gentle ping @HelixHEX.

**Comment by kyrylo** [comment]
@HelixHEX hey there. I did some research and it looks like we already provide typings. `@airbrake/browser` is a TypeScript project and our `dist` contains `.d.ts` files. When you install the library, check `node_modules/@airbrake/browser/dist`.
Does this PR improve the situation in any way? Basically, do we really need to introduce this change?

**Comment by HelixHEX** [comment]
@kyrylo 

That's strange because when I first tried installing the package I was getting errors saying that the typing's don't exist and it needs to be installed or manually typed. I'll try installing in a fresh package to see if the errors go away.

**Comment by kyrylo** [comment]
Thanks, looking forward to hear from you ☺️

**Comment by kyrylo** [comment]
Thanks for bringing this up. I am going to close this for now. Feel free to ping me when you have time.

**Pull request closed without merging**